AEPMessaging/Sources/MessagingRulesEngine.swift

```diff
@@ -18,19 +18,26 @@ import Foundation
 class MessagingRulesEngine {
     let rulesEngine: LaunchRulesEngine
     let runtime: ExtensionRuntime
+    let cache: Cache
+    let messagingCacheName = "com.adobe.messaging.cache"
+    let cachedMessagesName = "messages"
+    let cachedMessagesDelimiter = "||"
 
     /// Initialize this class, creating a new rules engine with the provided name and runtime
     init(name: String, extensionRuntime: ExtensionRuntime) {
         runtime = extensionRuntime
         rulesEngine = LaunchRulesEngine(name: name,
                                         extensionRuntime: extensionRuntime)
+        cache = Cache(name: messagingCacheName)
+        loadCachedMessages()
```

Comment:
it looks like this is the only location that loadCachedMessages is called from. would we want to check if cached messages are present and load them at every app launch?